### DISCUSSION TITLE
Customizing UI

### DISCUSSION THREAD

**Discussion by yodaphone** [amidaware/tacticalrmm#2087] [General]
Hi

Is there a way to customiza what appears in the top left of the Dashboard. For. e.g. can I change "TacticalRMM" to something custom ?

In MeshCentral you can do this in the config.json. I tried changing the "title" & "title2" but it doesnt reflect the change when i run the  mesh page

**Comment by silversword411** [comment]
Not yet
https://docs.tacticalrmm.com/roadmap/
